src/commands/githubActions/endToEndTest/index.ts

```diff
@@ -0,0 +1,88 @@
+import { CommandModule } from "yargs";
+import { CliGlobalOptions } from "../../../types.js";
+import { ensureDappnodeEnvironment } from "./ensureDappnodeEnvironment.js";
+import { readCompose, readManifest } from "../../../files/index.js";
+import { buildHandler } from "../../build.js";
+import { localDappmanagerTestApiUrl, localIpfsApiUrl } from "./params.js";
+import { executeTests } from "./executeTests.js";
+import { DappmanagerTestApi } from "./dappmanagerTestApi.js";
+
+interface CliCommandOptions extends CliGlobalOptions {
+  healthCheckUrl?: string;
+  errorLogsTimeout: number;
+  environmentByService?: string;
+}
+
+export const endToEndTest: CommandModule<
+  CliGlobalOptions,
+  CliCommandOptions
+> = {
+  command: "test-end-to-end",
+  describe: "Run end to end tests (Install from scratch and update)",
+  builder: {
+    healthCheckUrl: {
+      type: "string",
+      describe:
+        "Optional health check URL, if the HTTP code is not 200, the test will fail"
+    },
+    errorLogsTimeout: {
+      describe:
+        "Timeout in seconds to wait for error logs to appear. If error logs appear after the timeout, the test will fail",
+      type: "number",
+      default: 5
```

Comment:
I would say the default value should be higher as it is normal for the main process of the containers to take some time to start working properly